baekjoon/피보나치-수-5.js

```diff
@@ -0,0 +1,10 @@
+const n = parseInt(require("fs").readFileSync("/dev/stdin").toString());
+
+function solution(n) {
+    for (let i = 1; i <= n; i++) {
+        n = solution(n - 1) + solution(n - 2);
```

Comment:
```suggestion
    return solution(n - 1) + solution(n - 2);
```
재귀함수를 이용하면 for문과 같이 반복문을 이용하지 않아도 실행이 가능합니다.
여기서 f(0) = 0 인 것과 f(1) = 1인 경우만 조건문으로 return 해주면 for 문 없이도 가능 할 것 같아요.